plugins/modules/proxmox_backup_schedule.py

```diff
@@ -0,0 +1,264 @@
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+module: proxmox_backup_schedule
+
+short_description: Scheduling VM backups and removing them.
+
+version_added: 10.3.0
+
+description: The module modifies backup jobs such as set or delete C(vmid).
+
+author:
+  - "Marzieh Raoufnezhad (@raoufnezhad) <raoufnezhad@gmail.com>"
+  - "Maryam Mayabi (@mmayabi) <mayabi.ahm@gmail.com>"
+
+options:
+  vm_name:
+    description:
+      - The name of the Proxmox VM.
+      - Mutually exclusive with O(vm_id).
+    type: str
+  vm_id:
+    description:
+      - The ID of the Proxmox VM.
+      - Mutually exclusive with O(vm_name).
+    type: str
+  backup_id:
+    description: The backup job ID.
+    type: str
+  state:
+    description:
+        - If V(present), the module will update backup job with new VM ID.
+        - If V(absent), the module will remove the VM ID from all backup jobs where the VM ID has existed.
+    required: true
+    choices: ["present", "absent"]
+    type: str
+
+extends_documentation_fragment:
+  - community.general.proxmox.documentation
+  - community.general.attributes
+  - community.general.attributes.info_module
+  - community.general.proxmox.actiongroup_proxmox
+"""
+
+EXAMPLES = """
+- name: Scheduling VM Backups based on VM name
+  proxmox_backup_schedule:
+    api_user: 'myUser@pam'
+    api_password: '*******'
+    api_host: '192.168.20.20'
+    vm_name: 'VM Name'
+    backup_id: 'backup-b2adffdc-316e'
+    state: 'present'
+
+- name: Scheduling VM Backups based on VM ID
+  proxmox_backup_schedule:
+    api_user: 'myUser@pam'
+    api_password: '*******'
+    api_host: '192.168.20.20'
+    vm_id: 'VM ID'
+    backup_id: 'backup-b2adffdc-316e'
+    state: 'present'
+
+- name: Removing backup setting based on VM name
+  proxmox_backup_schedule:
+    api_user: 'myUser@pam'
+    api_password: '*******'
+    api_host: '192.168.20.20'
+    vm_name: 'VM Name'
+    state: 'absent'
+
+- name: Removing backup setting based on VM ID
+  proxmox_backup_schedule:
+    api_user: 'myUser@pam'
+    api_password: '*******'
+    api_host: '192.168.20.20'
+    vm_id: 'VM ID'
+    state: 'absent'
+
+- name: Removing backup setting based on VM name from specific backup job
+  proxmox_backup_schedule:
+    api_user: 'myUser@pam'
+    api_password: '*******'
+    api_host: '192.168.20.20'
+    vm_name: 'VM Name'
+    backup_id: 'backup-b2adffdc-316e'
+    state: 'absent'
+
+- name: Removing backup setting based on VM ID from specific backup job
+  proxmox_backup_schedule:
+    api_user: 'myUser@pam'
+    api_password: '*******'
+    api_host: '192.168.20.20'
+    vm_id: 'VM ID'
+    backup_id: 'backup-b2adffdc-316e'
+    state: 'absent'
+"""
+
+RETURN = """
+---
+backup_schedule:
+  description:
+    - If V(present), the backup_schedule will return True after adding the VM ID to the backup job.
+    - If V(absent), the backup_schedule will return True after removing it from the backup job.
+  returned: always, but can be empty
+  type: raw
```

Comment:
If the return value is a boolean, use `type: bool`. From the description I'm not sure what it actually is, though. Does this basically indicate whether the module made a change? That should be indicated by `changed`, and not also by an extra return value with the same value.